src/renderer/state.ts

```diff
@@ -353,6 +356,20 @@ export class AppState {
     this.isUpdatingElectronVersions = false;
   }
 
+  @action public hideChannels(channels: Array<ElectronReleaseChannel>) {
+    this.channelsToShow = this.channelsToShow.filter(
+      (ch) => !channels.includes(ch),
+    );
+  }
+
+  @action public showChannels(channels: Array<ElectronReleaseChannel>) {
+    const s = new Set<ElectronReleaseChannel>([
+      ...this.channelsToShow,
+      ...channels,
+    ]);
+    this.channelsToShow = [...s.values()];
+  }
+
```

Comment:
Can we use these functions in the `settings-electron.tsx` logic?

Reply:
Sure! aff8b380fb573e1ac93300276142b4d0b1d1c289